src/Client/Invoice.php

```diff
@@ -50,7 +50,7 @@ public function createInvoice(
 
         $body = json_encode(
             [
-            'amount'   => $amount->__toString(),
+            'amount' => !is_null($amount) ? $amount->__toString() : null,
```

Comment:
Please don't use `is_null()` but use ` === null`. Function calls have more overhead so they're considered bad practise.

Reply:
Changed it.